corehq/apps/app_manager/models.py

```diff
@@ -3786,7 +3786,7 @@ def root_module_id(self):
     def get_suite_forms(self):
         if not self.source_module:
             return []
-        return self.source_module.get_forms()
+        return (f for f in self.source_module.get_forms() if f.unique_id not in self.excluded_form_ids)
```

Comment:
You make an excellent point. I like generators, but you do need to know how they will be used (or indicate in the function name that you're going to get a generator), and they don't offer benefits on small data sets like this anyway.